Makefile

```
@@ -48,22 +48,36 @@ $(crossplane_sentinel): kind-setup csi-host-path-setup load-comp-image
 stackgres-setup: export KUBECONFIG = $(KIND_KUBECONFIG)
 stackgres-setup: $(crossplane_sentinel) ## Install StackGres
 	helm repo add stackgres-charts https://stackgres.io/downloads/stackgres-k8s/stackgres/helm/
-	helm upgrade --install --version 1.5.0 --create-namespace --namespace stackgres stackgres-operator  stackgres-charts/stackgres-operator \
-	--values stackgres/values.yaml
+	helm upgrade --install --version 1.7.0 --create-namespace --namespace stackgres stackgres-operator  stackgres-charts/stackgres-operator --values stackgres/values.yaml --wait
+	kubectl -n stackgres wait --for condition=Available deployment/stackgres-operator --timeout 120s
+
+	# wait max 60 seconds for secret to be created - it takes little bit longer now for secret to appear, therefore we need a mechanism to block execution until it appears
+	@for i in $$(seq 1 60); do \
+        if kubectl get secret stackgres-restapi-admin -n stackgres > /dev/null 2>&1; then \
+            echo "Secret found!"; \
+            break; \
+        else \
+            echo "Secret not found. Retrying ($$i/60)..."; \
+            sleep 1; \
+        fi; \
+	done;
```

I would not do so many "useless" echos:
```suggestion
	# wait max 60 seconds for secret to be created - it takes little bit longer now for secret to appear, therefore we need a mechanism to block execution until it appears
	echo "waiting for stackgres-restapi-admin secret creation..."
	@for i in $$(seq 1 60); do \
        if kubectl get secret stackgres-restapi-admin -n stackgres > /dev/null 2>&1; then \
            break; \
        else \
            sleep 1; \
        fi; \
	done;
```